app/views/citationInfo.html

```diff
@@ -15,7 +15,7 @@
             <th>Violations</th>
           </thead>
           <tbody>
-          <tr class="selectableRow" ng-repeat="citation in ctrl.citations" ng-class="{'selectedRow' : ctrl.citationSelected(citation)}" ng-click="ctrl.selectCitation(citation)">
+          <tr class="selectableRow" ng-repeat="citation in ctrl.citations" ng-class="{'selectedRow' : ctrl.citationSelected(citation)}" ng-click="ctrl.selectCitation(citation); ctrl.scrollTo('selectedCitationInfo');">
```

Comment:
Hmmm....this makes the interactions less testable. Please just keep the single "ctrl.selectCitation" action and have the anchorScroll call made from within that method. Then you can remove ctrl.scrollTo completely.